packages/fx-core/src/core/tools.ts

```diff
@@ -118,6 +119,24 @@ export function isValidProject(workspacePath?: string): boolean {
   }
 }
 
+export function getActiveEnv(projectRoot: string): string | undefined {
```

Comment:
Since you return `undefined`, please make sure you will not reference an `undefined` value before using the active env returned by this function.

Reply:
Yes, it makes sense. Added error handling